### ISSUE TITLE
Curious about the time cost to release a new pretrain model

### ISSUE THREAD

**Issue by Huuush** [open-mmlab/mmdetection#4776]
Since I use VFNet to train deepfashion2（190k train imgs） for cloth detection in slurm, Using 2 nodes,  2 GPUs / per node, and it takes me amost 15 days to finish 12 epochs. I just wanna to confirm whether I did something wrong. Thanks.

**Comment by hhaAndroid** [comment]
I suggest you refer to the model training log we provide. The Page is [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/vfnet/README.md)

**Comment by Huuush** [comment]
> I suggest you refer to the model training log we provide. The Page is [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/vfnet/README.md)

Thanks for replying. I got it!

**Comment by Huuush** [comment]
sry to bother. but I wanna to kown, according to the docs in mmdetection, "The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8*2 = 16)"  and it is set to 0.01. Now, if i  want to use the pretrain model, given by mmdet, to train on my custom dataset, how do i set lr? if my batch_size is also 8*2=16. Thx!
